Source/BootloaderToken.cpp

```diff
@@ -0,0 +1,11 @@
+#include "device.h"
+#include "BootloaderToken.h"
+
+__attribute__ ((section (".bootloader_token")))
+__attribute__ ((used))
+static const struct BootloaderToken bootloader_token {
+    BOOTLOADER_MAGIC,
+    HARDWARE_ID,
+    (BOOTLOADER_VERSION_MAJOR << 16) | BOOTLOADER_VERSION_MINOR,
```

Comment:
How about storing version as a string instead, like we do with the firmware?

Reply:
Done